test/pages/HaveTheGoodsBeenSubjectToLegalChallengesPageSpec.scala

```diff
@@ -28,4 +31,18 @@ class HaveTheGoodsBeenSubjectToLegalChallengesPageSpec extends PageBehaviours {
 
     beRemovable[Boolean](HaveTheGoodsBeenSubjectToLegalChallengesPage)
   }
+  "cleanup" - {
+    "should reset DescribeTheLegalChallengesPage" - {
+      "when HaveTheGoodsBeenSubjectToLegalChallengesPage is changed to No" in {
+        val emptyUserAnswers = UserAnswers("id", DraftId(1))
+
+        val ua = emptyUserAnswers
+          .set(DescribeTheLegalChallengesPage, "legalities")
+          .get
+
+        HaveTheGoodsBeenSubjectToLegalChallengesPage.cleanup(Some(false), ua) mustBe
+          Success(emptyUserAnswers)
+      }
+    }
```

Comment:
We want to be testing all scenarios here, namely what happens when the cleanup is called with a None, and Some(true).